src/tools/clippy/clippy_lints/src/dereference.rs

```diff
@@ -869,7 +870,8 @@ impl TyCoercionStability {
                 | ty::Int(_)
                 | ty::Uint(_)
                 | ty::Array(..)
-                | ty::Float(_)
+                | ty::Pat(..)
+            | ty::Float(_)
```

Comment:
```suggestion
                | ty::Float(_)
```
minor alignment fix that I'm surprised rustfmt didn't take care of